repository/a.json

```diff
@@ -2200,6 +2200,19 @@
 				}
 			]
 		},
+		{
+			"name": "Auto Dark mode for Linux",
+			"details": "https://github.com/smac89/autodark-sublime-plugin",
+			"labels": ["color scheme", "theme", "xdg-desktop-portal"],
+			"releases": [
+				{
+					"sublime_text": ">=4050",
+					"tags": true,
+					"dependencies": ["sublime_lib"],
```

Comment:
Not sure you'd want to lean on that project if you develop for ST4. Having dependencies also prevents you from using python 3.8. If you're mainly using it to interface with settings, I would forego the dependency and use the native api. 

Also, better to remove the dependencies key here. You have a dependencies.json in repo, that will be used to resolve the dependencies. As the documentation states, use or other not, not both, and the json is more convenient.